content/blocks/crafting/adv-uranium-refinery.hjson

```diff
@@ -0,0 +1,34 @@
+type: GenericCrafter
+name: Advance Uranium Refinery
+description: "Refine depleted uranium in a large batch, making it more active again."
+health: 750
+size: 3
+requirements:
+[
+    steel/75
```

Comment:
_You're processing uramium,but not playing with muds._
Just increase the amount.The amout of steels produced per minute at the early stage of the game should be around 500 (4*full-used steelworks).